GenericModConfigMenu/Api.cs

```diff
@@ -52,8 +52,22 @@ public interface IApi
 
     public class Api : IApi
     {
+        internal static Action<string, LogLevel> ApiLog;
         public void RegisterModConfig(IManifest mod, Action revertToDefault, Action saveToFile)
         {
+            if (mod is null) {
+                ApiLog("mod is null! Who called me??", LogLevel.Error);
+                throw new ArgumentNullException(nameof(mod));
+                }
+            if (revertToDefault is null) {
+                ApiLog($"{mod.UniqueID} gave null revertToDefault!", LogLevel.Error);
+                throw new ArgumentNullException(nameof(revertToDefault));
+                }
+            if (saveToFile is null) {
+                ApiLog($"{mod.UniqueID} gave null saveToFile!", LogLevel.Error);
+                throw new ArgumentNullException(nameof(saveToFile));
+                }
+
```

Comment:
This adds additional info if we get more of those "object reference is null" calls involving GMCM

Reply:
And I just realized I forgot to edit the brace formatting, sorry 😅